commands/accounts/remove.ts

```diff
@@ -16,16 +16,16 @@ const { loadAndValidateOptions } = require('../../lib/validation');
 
 const i18nKey = 'commands.accounts.subcommands.remove';
 
-exports.command = 'remove [--account]';
+exports.command = 'remove <account>';
```

Comment:
Does this make account required? I think it should be optional right since we fall back to a prompt?

Reply:
It sure does 😅, for some reason I thought we weren't prompting.  I'll revert this.